pennylane/bose/__init__.py

```diff
@@ -1,3 +1,17 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 """A module containing utility functions and mappings for working with bosonic operators. """
 
-from .bosonic import BoseWord, BoseSentence
+from .bosonic import BoseSentence, BoseWord
+from .bosonic_mapping import unary_mapping
```

Comment:
Something going on with the imports.
I get errors for `qml.unary_mapping`. This may be fixed when the binary mapping PR is merged as it imports `bose` into the `pennylane/__init__`